README.md

```diff
@@ -1,15 +1,17 @@
-# SIG: Web - website2017 - an amazing project
-This is [CompSoc Edinburgh](https://comp-soc.com)'s website, built by SIG: Web.
+# SIG: Web - website2020 (up to date!) - a super duper amazing project
+This is [CompSoc Edinburgh](https://comp-soc.com)'s website, built by SIG:Web.
 
-## How to test
-It uses Jekyll. It also works nicely with Docker.
+## Running on your computer
+It uses Jekyll. It also works very amazingly nicely with Docker.
 
-If you use Linux or macOS, run `docker-run.sh`. It will automatically build, server, and watch the directory for changes. It will listen on port 4000.
+If you use Linux or macOS, run `docker-run.sh`. It will automatically build, server, and watch the directory for changes and it will listen on port 4000. 
 
-If you're on Windows, please contribute some instructions. You'll hopefully be able to read `docker-run.sh` and adapt the commands to play nice with docker-machine, or native Docker if that's your thing.
+If you're on Windows (I'm sorry), please contribute some instructions. You'll hopefully be able to read `docker-run.sh` and adapt the commands to play nice with docker-machine, or native Docker if that's your thing.
```

Comment:
maybe we can offer to send flowers or chocolates

Reply:
Does this maybe read a bit better?
```suggestion
If you're on Windows, reconsider your life decisions.
```